.github/workflows/sonar_analysis.yml

```
@@ -0,0 +1,75 @@
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=3
+          --env sonar.jdbc.username=sonar
+          --env sonar.jdbc.password=sonar
+          --env sonar.jdbc.url=jdbc:postgresql://postgres/sonarqube
+          --memory=2g
+          --cpus=2
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)